graylog2-web-interface/src/theme/GraylogThemeProvider.jsx

```diff
@@ -37,13 +37,12 @@ const createTheme = (mode, themeColors, changeMode): ThemeInterface => {
   };
 };
 
-const GraylogThemeProvider = ({ children }: Props) => {
-  const colorScheme = usePrefersColorScheme();
-  const [mode, setCurrentThemeMode] = useCurrentThemeMode(colorScheme);
+const GraylogThemeProvider = ({ children, defaultMode }: Props) => {
+  const [mode, setCurrentThemeMode] = defaultMode ? [defaultMode, () => {}] : useCurrentThemeMode();
```

Comment:
IMO this is a no-go for hooks, which should always be called unconditionally. Instead, we could pass in the `defaultMode` and use it accordingly.